index.php

```diff
@@ -123,7 +123,7 @@
 		$logout_limit = 1440;
 	}
 	else if($player->forbidden_seal) {
-		$logout_limit *= 2;
+        $logout_limit = System::$premium_benefits[json_decode($player->forbidden_seal, true)['level']]['logout_timer'];
```

Comment:
Have you tested this with `forbidden_seal` being empty? (just a blank string, `""`)

To me it seems like it would crash, and we need to add error handling for the empty case

Reply:
This doesn't occur, locally anyway. It just skips over the statement for both newly created accounts and accounts with expired seals.